docs/scipp-neutron/from-mantid-to-scipp.ipynb

```diff
@@ -547,6 +547,52 @@
     "histo = sc.histogram(events, edges)"
    ]
```

Comment:
Not changed here, but at top of notebook says "(other rather conversion to scipp)" doesn't make sense.

Reply:
What part does not make sense? Would a colon after "rather" help in readability?

Reply:
Says "other rather" at moment. My guess is that it should say "(or rather conversion to scipp)".